chapter_11_ansible.asciidoc

```diff
@@ -874,12 +941,24 @@ on the server:
 ----
 ====
 
+// CSANAD: I would remind the reader we need to run ansible-playbook again.
+
 That gets us to
 
 ----
 selenium.common.exceptions.NoSuchElementException: Message: Unable to locate
 element: [id="id_list_table"]; [...]
 ----
+// CSANAD: I have a different error:
+//         AssertionError: 'To-Do' not found in 'Bad Request(400)'
+//
+// >>> from django.conf import settings
+// >>>
+// >>>
+// >>> settings.ALLOWED_HOSTS
+// ['"192.168.122.23"']
```

Comment:
I tried a few things to remove the double quotes but somehow it's always there. I'll get back to this problem later. All ideas are welcome.